### PR TITLE
Add Deno announcement blog

### PR DESCRIPTION
**Pull request by ebenezerdon**
## What does this PR do?

- Adds Deno announcement blog
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/deno-runtime-announcment`
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by adityaoberai** [comment]
@TorstenDittmann I just noticed that our Coolify deployment showed as failed but a functioning link is available in the logs: http://1417.gkswsco.37.27.92.31.sslip.io

Not sure why this is occurring but we should look into it

File: src/routes/blog/post/deno-runtime-announcment/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+It’s easy to begin building with Deno on Appwrite Cloud. Here’s how:
+
+1. **Create a new function** in your Appwrite Cloud project using Deno Quick Start.
+2. **Connect git repository**, or choose to connect later if you just want to see it in action.
+3. **Visit your function’s domain** to make your first execution.
+4. **Implement your function** and benefit from Deno’s amazing performance and tooling features, all within the powerful Appwrite ecosystem.
+
+Supporting Deno opens up new possibilities for Appwrite developers, empowering you to build faster and more securely while taking advantage of Deno’s cutting-edge features. And with the recent release of Deno 2.0, now is the perfect time to explore what this exciting runtime has to offer.
+
+We’re excited to see what you’ll create with Deno on Appwrite Cloud.
```
Comment (adityaoberai) [inline src/routes/blog/post/deno-runtime-announcment/+page.markdoc:53]: @ebenezerdon should we not link to our docs at all in this section?

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@ebenezerdon just 1 comment

File: src/routes/blog/post/deno-runtime-announcment/+page.markdoc (hunk 1 above)
Comment (ebenezerdon) [reply src/routes/blog/post/deno-runtime-announcment/+page.markdoc:53]: I think we should. Should I update this @LauraDuRy?

Comment (ebenezerdon) [reply src/routes/blog/post/deno-runtime-announcment/+page.markdoc]: > @ebenezerdon should we not link to our docs at all in this section?

Just updated!

File: src/routes/blog/post/deno-runtime-announcment/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The runtime also benefits from Deno’s comprehensive toolset, including its built-in linter, formatter, tester, dependency inspector, [and more](https://docs.deno.com/runtime/reference/cli/all_commands/). Reducing reliance on third-party tooling streamlines workflow and minimizes external dependencies. This means fewer headaches when it comes to maintenance and more focus on writing the code that matters.
+
+# **The Deno 2.0 advantage**
+
+Our support for Deno comes at an especially exciting time with the recent release of **Deno 2.0**. This version introduces substantial improvements, making the integration even more powerful for Appwrite developers:
+
+- **Improved Node.js compatibility**: Deno 2.0 introduces the process global variable, which allows for better interoperability with existing Node.js codebases.
+- **Enhanced dependency Management**: Deno 2.0 is fully compatible with NPM, and introduces monorepo support to fit needs of large codebases.
```
Comment (adityaoberai) [inline src/routes/blog/post/deno-runtime-announcment/+page.markdoc:37]:
```suggestion
- **Enhanced dependency management**: Deno 2.0 is fully compatible with NPM, and introduces monorepo support to fit needs of large codebases.
```

File: src/routes/blog/post/deno-runtime-announcment/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- **Upgraded TypeScript support**: With TypeScript 5.6 included, Deno 2.0 ensures you’re working with the latest features and optimizations in the language.
+- **Upgraded security**: The updated permission system in Deno 2.0 adds more granular control over application access, including the introduction of Deno.errors.NotCapable to distinguish between system and Deno-specific issues, providing a clearer security model for your projects.
+- **Better web framework support**: Deno 2.0 now works with well-known meta frameworks such as Next.js, Remix, Svelte, Qwik, and more. Support for those, as well as typical web toolings like Vite, Parcel, and Rollup, all make Deno 2.0 a great candidate for building modern web applications.
+
+# **How to get started with Deno on Appwrite Cloud**
+
+It’s easy to begin building with Deno on Appwrite Cloud. Here’s how:
+
+1. **Create a new function** in your Appwrite Cloud project using Deno Quick Start. Here's our Functions [quickstart guide](https://appwrite.io/docs/products/functions/quick-start).
+2. **Connect git repository**, or choose to connect later if you just want to see it in action.
```
Comment (adityaoberai) [inline src/routes/blog/post/deno-runtime-announcment/+page.markdoc:47]:
```suggestion
2. **Connect GitHub repository**, or choose to connect later if you just want to see it in action.
```

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@ebenezerdon just a couple of other minor edits and we're good to go

**Review by LauraDuRy — APPROVED** [review]

**Review by LauraDuRy — APPROVED** [review]

**Pull request merged**